code/datums/components/bloodysoles.dm

```diff
@@ -1,3 +1,5 @@
+/// The percentage of blood we lose on each step
+#define BLOOD_PERCENT_LOSS_ON_STEP 0.25
```

Comment:
When I was tweaking this for another project, I went with `0.33`, and found it to be a pretty happy range. I suggest moving it up to that.

Reply:
done